### PR TITLE
chore(build): add grace test runs to daily PR checks

### PR DESCRIPTION
**Pull request by gshif**
Closes #

Describe your proposed changes here.

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Feature flagged (if modified API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -210,6 +210,18 @@ jobs:
...
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run:
+          command: ./bin/linux/influxd --store=memory --log-level=debug
+          background: true
+      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/grace/test-results/grace-results quay.io/influxdb/grace:daily
+      - store_artifacts:
+          path: ~/project
```
Comment (danxmoran) [inline .circleci/config.yml:224]: Should we add this block:
```yaml
- store_test_results:
    path: ~/project
```
To match `grace_nightly`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -210,6 +210,18 @@ jobs:
...
+  grace_daily:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run:
+          command: ./bin/linux/influxd --store=memory --log-level=debug
+          background: true
+      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/grace/test-results/grace-results quay.io/influxdb/grace:daily
```
Comment (danxmoran) [inline .circleci/config.yml:222]: Is `--net host` the flag that enables the Docker image to see the `influxd` instance running in the background?

File: .circleci/config.yml (hunk 1 above)
Comment (gshif) [reply .circleci/config.yml:224]: Actually we should remove the `store_test_results:` from grace_nightly.The results are being uploaded by `store_artifact` step

**Review by danxmoran — APPROVED** [review]
I see that it ran & passed on this PR, LGTM

File: .circleci/config.yml (hunk 2 above)
Comment (gshif) [reply .circleci/config.yml:222]: The short answer is yes

**Pull request merged**